### PR TITLE
Wut object recognition

### PR DESCRIPTION
**Pull request by maciek-slon**
Finished object_detection services. 

What is done:
- package description on wiki: https://github.com/rapp-project/rapp-platform/wiki/RAPP-Object-Recognition
- web services
- unit, functional and integration tests
- python API calls
- code documentation

Some API parts are missing at the moment (C++, JS), but platform part is ready to be merged.

Related pull request for RappAPI: https://github.com/rapp-project/rapp-api/pull/48

I'm closing #174 as out-of-date.

### CODE REVIEW

File: rapp_web_services/services/object_detection_clear_models/svc.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+/**
+ *  [Object Recognition - Clear Models]
+ *  Handles requests to object_detection/clear_models RAPP Platform Service
+ *
+ *  Service Implementation.
+ */
+function svcImpl(req, resp, ros) {
+  var rosMsg = new interfaces.ros_req();
+  rosMsg.user = req.body.user;
```
Comment (klpanagi) [inline rapp_web_services/services/object_detection_clear_models/svc.js]: Retrieve `user` from req object. A `username` property is applied to the `req` object after successful authentication.

``` js
var user = req.username
```

**Comment by maciek-slon** [comment]
Username removed from service calls.

**Comment by etsardou** [comment]
Status?

**Comment by athallas** [comment]
Hey @maciek-slon! Is this ready to be merged?

**Comment by maciek-slon** [comment]
Yup, it is ready.

**Pull request merged**